README.md

```diff
@@ -50,11 +50,11 @@ Feel free to create an issue if you need help. However, you might find it easier
 
 [![PolyMC Discord](https://img.shields.io/discord/923671181020766230?label=PolyMC%20Discord)](https://discord.gg/xq7fxrgtMP)
 
-For people who don't want to use Discord, we have a Matrix Space which is bridged to the Discord server. Be sure to enable spaces first (Settings -> Labs -> Spaces), and then you may join the space:
+For people who don't want to use Discord, we have a Matrix Space which is bridged to the Discord server.
 
 [![PolyMC Space](https://img.shields.io/matrix/polymc:polymc.org?label=PolyMC%20Space&server_fqdn=matrix.polymc.org)](https://matrix.to/#/#polymc:polymc.org)
 
-Matrix's support for spaces is still in development, so if you have issues accessing rooms via the space, then you can join the rooms directly:
+If there is any issues with the space or you are using a client that does not support the feature here are the individual rooms:
```

Comment:
ditto

Reply:
Matrix spaces is no longer under development its out of beta and a full feature

Reply:
Some clients however dont support it and makes it more clesr

Reply:
Ok, switch that "is" to "are" and I can squash merge.